Svc/SeqDispatcher/CMakeLists.txt

```diff
@@ -0,0 +1,34 @@
+####
+# F prime CMakeLists.txt:
+#
+# SOURCE_FILES: combined list of source and autocoding files
+# MOD_DEPS: (optional) module dependencies
+# UT_SOURCE_FILES: list of source files for unit tests
+#
+####
+set(SOURCE_FILES
+  "${CMAKE_CURRENT_LIST_DIR}/SeqDispatcher.fpp"
+  "${CMAKE_CURRENT_LIST_DIR}/SeqDispatcher.cpp"
+)
+
+# Uncomment and add any modules that this component depends on, else
+# they might not be available when cmake tries to build this component.
+
+set(MOD_DEPS
+    common/types
+    Svc/CmdSequencer
+)
+
+register_fprime_module()
+
+# Register the unit test build
+set(UT_SOURCE_FILES
+  "${CMAKE_CURRENT_LIST_DIR}/SeqDispatcherCommands.fppi"
```

Comment:
Typically you don't need to add `.fppi` files to source lists.  Is this a workaround for the version I have?

Reply:
Not sure if we've tried without including the fppi files. Will try removing if possible

Reply:
Removed redundant fppi in source list